### PR TITLE
zh-trans: add content/zh/blog/_posts/2018-10-15-steering-election-results.md

### PR DESCRIPTION
**Pull request by SataQiu**

### CODE REVIEW

File: content/zh/blog/_posts/2018-10-15-steering-election-results.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+* 本次选举由康奈尔大学主办 [CIVS](https://civs.cs.cornell.edu/)！
+
+<!--
+## Get Involved with the Steering Committee
+-->
+## 加入督导委员会
+<!--
+You can follow along to Steering Committee [backlog items](https://git.k8s.io/steering/backlog.md) and weigh in by filing an issue or creating a PR against their [repo](https://github.com/kubernetes/steering). They meet bi-weekly on [Wednesdays at 8pm UTC](https://github.com/kubernetes/steering) and regularly attend Meet Our Contributors.
+-->
+你可以关注督导委员会的[积压项目](https://git.k8s.io/steering/backlog.md)，并通过提交 issue 或针对他们的[代码仓库](https://github.com/kubernetes/steering)创建 PR 来参与。他们在[UTC 时间每周三晚 8 点](https://github.com/kubernetes/steering)举行会议，并定期与我们的贡献者见面。
```
Comment (chenrui333) [inline content/zh/blog/_posts/2018-10-15-steering-election-results.md]: backlog 算是 agile 里面的一个术语，应该是`任务`或者`需求`的意思
```suggestion
你可以关注督导委员会的[积压项目](https://git.k8s.io/steering/backlog.md)，并通过提交 issue 或针对他们的[代码仓库](https://github.com/kubernetes/steering)创建 PR 来参与。他们在[UTC 时间每周三晚 8 点](https://github.com/kubernetes/steering)举行会议，并定期与我们的贡献者见面。
```

File: content/zh/blog/_posts/2018-10-15-steering-election-results.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Steering Committee Meetings:
+-->
+督导委员会会议：
+
+* [YouTube 播放列表](https://www.youtube.com/playlist?list=PL69nYSiGNLP1yP1B_nd9-drjoxp0Q14qM)
+
+<!--
+Meet Our Contributors Steering AMA’s: 
+-->
+与我们的贡献者会面：
```
Comment (chenrui333) [inline content/zh/blog/_posts/2018-10-15-steering-election-results.md]:
```suggestion
与我们的贡献者见面：
```

**Comment by SataQiu** [comment]
@chenrui333  Thanks, I need your approve! 😆

**Comment by fleeto** [comment]
`[trans-bot:N/A]`

Thank you @SataQiu, the [related task issue](https://github.com/k8smeetup/website-tasks/issues/267) has no assignee. will not be reported to the task issues

**Pull request merged**